StripeUICore/StripeUICore/Source/Elements/SubtitleElement.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SubtitleElement.swift
+//  StripeUICore
+//
+//  Created by Nick Porter on 3/26/25.
+//
+
+import UIKit
+
+@_spi(STP) public class SubtitleElement: Element {
+    public let view: UIView
+    public let collectsUserInput: Bool = false
+    weak public var delegate: ElementDelegate?
+
+    public init(view: UIView, isHorizontalMode: Bool) {
+        let padding = isHorizontalMode ? .zero : UIEdgeInsets(top: 0, left: 0, bottom: 20, right: 0)
+        guard padding != .zero else {
+            self.view = view
+            return
+        }
```

Comment:
Seems like we can have the guard statement before calculating padding?
```
guard !isHorizontalMode else {
    self.view = view
    return
}
let padding = UIEdgeInsets(top: 0, left: 0, bottom: 20, right: 0)
```

Reply:
yep can do!